src/CONST.ts

```diff
@@ -595,6 +596,11 @@ const CONST = {
                 },
             },
         },
+        RESOLUTIONS: {
+            INVITE: 'invited',
+            NOTHING: 'nothing',
+            UNRESOLVED: 'unresolved',
```

Comment:
Why do we have an `UNRESOLVED` resolution? This shouldn't be needed. If an actionable mention whisper doesn't have a `resolution` field, it's assumed to be unresolved.